frontend/packages/db-structure/src/schema/dbStructure.ts

```diff
@@ -43,13 +43,15 @@ const tableSchema = v.object({
 
 export type Table = v.InferOutput<typeof tableSchema>
 
+const cardinalitySchema = v.picklist(['ONE_TO_ONE', 'ONE_TO_MANY'])
```

Comment:
I see, I didn't know we can write that way 👍

Reply:
Conflict with this PR https://github.com/liam-hq/liam/pull/167 , please pull main branch 🙏🏻